src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -678,9 +736,7 @@ REDHAWK_PALEXPORT void PalPrintFatalError(const char* message)
 {
     // Write the message using lowest-level OS API available. This is used to print the stack overflow
     // message, so there is not much that can be done here.
-    // write() has __attribute__((warn_unused_result)) in glibc, for which gcc 11+ issue `-Wunused-result` even with `(void)write(..)`,
-    // so we use additional NOT(!) operator to force unused-result suppression.
-    (void)!write(STDERR_FILENO, message, strlen(message));
+    write(STDERR_FILENO, message, strlen(message));
```

Comment:
Incorrectly resolved confict

Reply:
Fixed (and rebased to make Blame show correct attribution for these lines)